search.php

```diff
@@ -96,7 +96,7 @@
 else if ($micromanager) {
 	$url = "http://micro-manager.3463995.n2.nabble.com/template/NamlServlet.jtp?macro=search_page&node=3463995&query=$q";
 }
-else if ($slim_curve) {
+else if ($flimlib) {
 	$url = "https://www.google.com/?gws_rd=ssl#q=site:loci.wisc.edu%2Fpipermail%2Fslim-curve+$q";
```

Comment:
I didn't change `slim-curve` to `flimlib` here, as is seems the Google search is meant to crawl the `slim-curve` archives. The search results I got (e.g. by searching for "slim") were leading to `404 Not found` pages though.